sdm/src/main/resources/cds/com.sap.cds/sdm/attachments-annotations.cds

```diff
@@ -0,0 +1,41 @@
+using {
+    sap.attachments.MediaData,
+    sap.attachments.Attachments
+} from './attachments';
+
+annotate MediaData with @UI.MediaResource: {Stream: content} {
+    content   @(
+        title                      : '{i18n>attachment_content}',
+        Core.MediaType             : mimeType,
+        ContentDisposition.Filename: fileName,
+        ContentDisposition.Type    : 'inline'
+    );
+    mimeType  @(
+        title: '{i18n>attachment_mimeType}',
+        Core.IsMediaType
+    );
+    fileName  @(title: '{i18n>attachment_fileName}');
+    contentId @(UI.Hidden: true);
+}
+
+annotate Attachments with @UI: {
+    HeaderInfo: {
+        $Type         : 'UI.HeaderInfoType',
+        TypeName      : '{i18n>attachment}',
+        TypeNamePlural: '{i18n>attachments}',
+    },
+    LineItem  : [
+        {Value: fileName},
+        {Value: content},
+        {Value: createdAt},
+        {Value: createdBy},
+        {Value: note}
+    ]
+} {
+    note       @(title: '{i18n>attachment_note}');
+    modifiedAt @(odata.etag);
+}
+
+annotate Attachments with @Common: {SideEffects #ContentChanged: {
```

Comment:
What is the significance of this annotation ?

Reply:
I will check this once my POC is done